--- e16-0.16.8.1/src/main.c~	2006-07-17 22:23:50.000000000 +0200
+++ e16-0.16.8.1/src/main.c	2006-07-17 22:28:30.000000000 +0200
@@ -487,6 +487,20 @@
 }
 
 static void
+RunMenuCheck(void)
+{
+
+   char                file[FILEPATH_LEN_MAX];
+
+   if (fork())
+      return;
+
+   Esnprintf(file, sizeof(file), "exec %s/scripts/e_check_menu", EDirRoot());
+   execl(usershell(getuid()), usershell(getuid()), "-c", (char *)file, NULL);
+   exit(0);
+}
+
+static void
 RunInitPrograms(void)
 {
    if (Mode.wm.session_start)
@@ -498,7 +512,8 @@
      {
 	RunMenuGen();
 	RunDocBrowser();
-     }
+     } else
+	RunMenuCheck();
 }
 
 const char         *
