diff -urd e16-0.16.8.1/src/config.c e16-0.16.8.1.edirconf/src/config.c
--- e16-0.16.8.1/src/config.c	2006-03-28 20:06:00.000000000 +0200
+++ e16-0.16.8.1.edirconf/src/config.c	2006-07-17 21:58:47.000000000 +0200
@@ -200,7 +200,7 @@
    def_shell = usershell(getuid());
 
    Esnprintf(execline, sizeof(execline), "%s " "-P " "-nostdinc " "-undef "
-	     "-include %s/config/definitions " "-I%s " "-I%s/config "
+	     "-include %s/definitions " "-I%s " "-I%s "
 	     "-D ENLIGHTENMENT_VERSION=%s " "-D ENLIGHTENMENT_ROOT=%s "
 	     "-D ENLIGHTENMENT_BIN=%s "
 	     "-D ENLIGHTENMENT_THEME=%s " "-D ECONFDIR=%s "
@@ -209,7 +209,7 @@
 	     "-D SCREEN_DEPTH_%i=1 " "-D USER_NAME=%s " "-D HOME_DIR=%s "
 	     "-D USER_SHELL=%s " "-D ENLIGHTENMENT_VERSION_015=1 "
 	     "%s %s",
-	     epp_path, EDirRoot(), Mode.theme.path, EDirRoot(),
+	     epp_path, EDirConf(), Mode.theme.path, EDirConf(),
 	     ENLIGHTENMENT_VERSION, EDirRoot(), EDirBin(),
 	     Mode.theme.path, EDirUser(), EDirUserCache(), VRoot.w,
 	     VRoot.h, VRoot.w, VRoot.h, VRoot.depth, def_user, def_home,
@@ -450,7 +450,7 @@
      }
 
    /* look in system config dir */
-   Esnprintf(s, sizeof(s), "%s/config", EDirRoot());
+   Esnprintf(s, sizeof(s), "%s", EDirConf());
    p = FindFileLocalized(file, s, localized);
    if (p)
       return p;
diff -urd e16-0.16.8.1/src/E.h e16-0.16.8.1.edirconf/src/E.h
--- e16-0.16.8.1/src/E.h	2006-03-28 20:06:01.000000000 +0200
+++ e16-0.16.8.1.edirconf/src/E.h	2006-07-17 21:59:16.000000000 +0200
@@ -825,6 +825,7 @@
 
 /* main.c */
 void                EExit(int exitcode);
+const char         *EDirConf(void);
 const char         *EDirRoot(void);
 const char         *EDirBin(void);
 const char         *EDirUser(void);
diff -urd e16-0.16.8.1/src/misc.c e16-0.16.8.1.edirconf/src/misc.c
--- e16-0.16.8.1/src/misc.c	2006-03-28 20:06:01.000000000 +0200
+++ e16-0.16.8.1.edirconf/src/misc.c	2006-07-17 21:56:16.000000000 +0200
@@ -147,6 +147,13 @@
    return k1 + x * l;
 }
 
+const char         *
+EDirConf(void)
+{
+   return ENLIGHTENMENT_CONF;
+}
+
+
 void
 ETimedLoopInit(int k1, int k2, int speed)
 {
