diff -urd e16-0.16.8.1/config/Makefile.am e16-0.16.8.1.edirconf/config/Makefile.am
--- e16-0.16.8.1/config/Makefile.am	2006-03-18 22:58:14.000000000 +0100
+++ e16-0.16.8.1.edirconf/config/Makefile.am	2006-07-17 22:09:56.000000000 +0200
@@ -1,6 +1,6 @@
 SUBDIRS = pix
 
-configdata_DATA = \
+sysconf_DATA = \
 actionclasses.cfg \
 backup-borders.cfg \
 backup-colormodifiers.cfg \
@@ -14,10 +14,10 @@
 menus.cfg \
 e16keyedit.db
 
-configdatadir = $(ENLIGHTENMENT_ROOT)/config
+configdatadir = $(sysconfdir)
 
 TRANSLATIONS = # config.ja config.pl config.ko
-EXTRA_DIST = $(configdata_DATA) $(TRANSLATIONS) strings.c
+EXTRA_DIST = $(sysconf_DATA) $(TRANSLATIONS) strings.c
 
 no-install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(configdatadir)
diff -urd e16-0.16.8.1/config/pix/Makefile.am e16-0.16.8.1.edirconf/config/pix/Makefile.am
--- e16-0.16.8.1/config/pix/Makefile.am	2003-07-15 07:46:24.000000000 +0200
+++ e16-0.16.8.1.edirconf/config/pix/Makefile.am	2006-07-17 22:06:17.000000000 +0200
@@ -1,4 +1,4 @@
-epixdir = $(ENLIGHTENMENT_ROOT)/config/pix
+epixdir = $(sysconfdir)/pix
 
 epix_DATA = wait1.png wait2.png wait3.png wait4.png \
             wait5.png wait6.png wait7.png wait8.png \
diff -urd e16-0.16.8.1/configure.in e16-0.16.8.1.edirconf/configure.in
--- e16-0.16.8.1/configure.in	2006-03-28 19:53:50.000000000 +0200
+++ e16-0.16.8.1.edirconf/configure.in	2006-07-17 22:11:57.000000000 +0200
@@ -30,7 +30,7 @@
   AC_DEFINE_UNQUOTED(ENLIGHTENMENT_RELEASE, "$ENLIGHTENMENT_RELEASE", [Release])
 fi
 
-ALL_LINGUAS="bg bs da de en_US es fr hu ko nl no ru sv pt pl pt_BR tr ja"
+ALL_LINGUAS="bg bs da de en_US es fr hu ja ko nb nl pl pt pt_BR ru sv tr"
 
 AC_PROG_CC
 AC_PROG_MAKE_SET
@@ -68,10 +68,6 @@
 AM_GNU_GETTEXT
 AM_GNU_GETTEXT_VERSION(0.12.1)
 
-AC_ARG_ENABLE(fsstd,
-  [  --enable-fsstd          install files following FSSTD @<:@default=yes@:>@],,
-  enable_fsstd=yes)
-
 AC_ARG_ENABLE(sound,
   [  --enable-sound          compile with sound support @<:@default=yes@:>@],,
   enable_sound=yes)
@@ -92,20 +88,16 @@
 dnl The following test must come after AM_GNU_GETTEXT as it relies on
 dnl ${DATADIRNAME} being defined.
 
-if test "x$enable_fsstd" = "xyes"; then
-  LOCALEDIR=${prefix}/${DATADIRNAME}/locale
-else
-  datadir=${prefix}
-  exec_prefix=${prefix}/e16
-  LOCALEDIR=${prefix}/e16/locale
-fi
 ENLIGHTENMENT_ROOT=`eval echo ${datadir}/e16`
 ENLIGHTENMENT_BIN=`eval echo ${bindir}`
+ENLIGHTENMENT_CONF=`eval echo ${sysconfdir}`
 AC_SUBST(ENLIGHTENMENT_ROOT)
 AC_SUBST(ENLIGHTENMENT_BIN)
+AC_SUBST(ENLIGHTENMENT_CONF)
 AC_SUBST(LOCALEDIR)
 AC_DEFINE_UNQUOTED(ENLIGHTENMENT_ROOT, "$ENLIGHTENMENT_ROOT", [The installation root directory])
 AC_DEFINE_UNQUOTED(ENLIGHTENMENT_BIN, "$ENLIGHTENMENT_BIN", [The installation bin directory])
+AC_DEFINE_UNQUOTED(ENLIGHTENMENT_CONF, "$ENLIGHTENMENT_CONF", [The installation config directory])
 AC_DEFINE_UNQUOTED(LOCALEDIR, "$LOCALEDIR",  [The installation locale directory])
 
 if test "x$enable_sound" = "xyes"; then
