--- enlightenment-0.16.999.038/src/modules/temperature/e_mod_main.c.orig	2007-03-26 01:28:22.000000000 +0200
+++ enlightenment-0.16.999.038/src/modules/temperature/e_mod_main.c	2007-09-09 19:21:22.754853908 +0200
@@ -286,6 +286,9 @@
 	      inst->sensor_path = evas_stringshare_add("/proc/omnibook/temperature");
 	      break;
 	   case SENSOR_TYPE_LINUX_MACMINI:
+	      if(!ecore_file_exists("/sys/devices/temperatures/cpu_temperature") && ecore_file_exists("/sys/devices/temperatures/sensor1_temperature"))
+		  inst->sensor_path = evas_stringshare_add("/sys/devices/temperatures/sensor1_temperature");
+	      else
 		  inst->sensor_path = evas_stringshare_add("/sys/devices/temperatures/cpu_temperature");
 	      break;
 	   case SENSOR_TYPE_LINUX_I2C:
