diff -u enlightenment-0.16.6/scripts.orig/e_gen_menu enlightenment-0.16.6/scripts/e_gen_menu
--- enlightenment-0.16.6/scripts.orig/e_gen_menu	Wed Nov  5 17:45:58 2003
+++ enlightenment-0.16.6/scripts/e_gen_menu	Thu Nov 13 11:55:59 2003
@@ -25,7 +25,7 @@
 ##############################################################################
 
 # Likely  prefixes
-$Prefixes = "/usr/local:/usr:/opt";
+$Prefixes = "/usr/local:/usr:/usr/X11R6:/opt";
 
 # Where to look for GNOME2/KDE2,3 stuff
 $AppDirs = MkDirList($Prefixes, "/share/applications");
